docs/source/uses.md

```diff
@@ -0,0 +1,26 @@
+## Interesting Uses
+
+These are just some of the interesting deployments by current users of the
+Jupyter Kernel Gateway:
+
+* Attach a local Jupyter Notebook server to a compute cluster in the cloud 
+  running near big data (e.g., interactive gateway to Spark)
+* Enable a new breed of non-notebook web clients to provision and use 
+  kernels (e.g., dashboards using 
+  [jupyter-js-services](https://github.com/jupyter/jupyter-js-services))
+* Scale kernels independently from clients (e.g., via 
+  [tmpnb](https://github.com/jupyter/tmpnb), [Binder](https://mybinder.org),
+  or your favorite cluster manager)
+* Create microservices from notebooks via 
+  [`notebook-http` mode](#notebook-http-mode)
+
```

Comment:
I'm a little worried about burying these in the doc. We meant them as aids for understanding what the heck you'd want to do with the KG after reading the very technical introduction which assumes a lot of knowledge about the underpinnings of Jupyter.